metadata-ingestion/src/datahub/ingestion/source/s3/source.py

```diff
@@ -477,6 +476,10 @@ def get_fields(self, table_data: TableData, path_spec: PathSpec) -> List:
     def get_table_profile(
         self, table_data: TableData, dataset_urn: str
     ) -> Iterable[MetadataWorkUnit]:
+        from pydeequ.analyzers import AnalyzerContext
```

Comment:
Can you add a comment here or somewhere else discussing why we do relative imports